<ui version="4.0" >
 <class>BezierGlyphBuilderClass</class>
 <widget class="QDialog" name="BezierGlyphBuilderClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>BezierGlyphBuilder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Input file name</string>
       </property>
       <property name="buddy" >
        <cstring>inputFileNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputFileNameLineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="loadButton" >
       <property name="text" >
        <string>&amp;Load</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="glyphLabel" >
       <property name="text" >
        <string>&amp;Glyph</string>
       </property>
       <property name="buddy" >
        <cstring>glyphComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="glyphComboBox" />
     </item>
     <item>
      <widget class="QLabel" name="variantLabel" >
       <property name="text" >
        <string>&amp;Variant</string>
       </property>
       <property name="buddy" >
        <cstring>variantComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="variantComboBox" />
     </item>
     <item>
      <widget class="QLabel" name="stepLabel" >
       <property name="text" >
        <string>&amp;Samples</string>
       </property>
       <property name="buddy" >
        <cstring>samplesSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="samplesSpinBox" >
       <property name="maximum" >
        <number>9999</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="glyphViewLayout" >
     <property name="leftMargin" >
      <number>1</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Output file name</string>
       </property>
       <property name="buddy" >
        <cstring>outputFileNameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="outputFileNameLineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="buildButton" >
       <property name="text" >
        <string>&amp;Build</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton" >
       <property name="text" >
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>glyphComboBox</tabstop>
  <tabstop>variantComboBox</tabstop>
  <tabstop>samplesSpinBox</tabstop>
  <tabstop>buildButton</tabstop>
  <tabstop>quitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>released()</signal>
   <receiver>BezierGlyphBuilderClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>577</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel" >
     <x>585</x>
     <y>462</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
